src/GoEngine.ts

```diff
@@ -492,8 +496,24 @@ export class GoEngine {
             unpackMoveTree(this.move_tree, config.move_tree);
         }
 
+        let removed;
         if (config.removed) {
-            let removed = this.decodeMoves(config.removed);
+            removed = this.decodeMoves(config.removed);
+        }
+        if (!this.territory_included_in_sgf &&
+            typeof(config.removed) === "undefined" &&
+            config.original_sgf &&
+            (/[0-9.]+/.test(self.outcome))) {
+            // Game data for SGF uploaded games don't always include removed stones, so we use score
+            // estimator to find probably dead stones to get a closer approximation of what
+            // territories should be marked in the final board position.
+            //
+            // NOTE: Some Go clients (not OGS, at least for now) do include dead stones in their
+            // SGFs, which we respect if present and skip using the score estimator.
+            let se = new ScoreEstimator(this.goban_callback, this, AUTOSCORE_TRIALS, AUTOSCORE_TOLERANCE);
```

Comment:
This line causes the server to kick off the ScoreEstimator on the server side every time an SGF is loaded without scoring, so we'd either need to check if we're in operating on the server and not run this, or simply remove this. I'm slightly on the side of not run this at all, for a couple of reasons: 1) if the sgf didn't set the territory, not running this is the "pure" way to see that SGF - and if the user wants the estimation they can always click the estimator button. 2) I'm hoping to ditch the score estimator all together in favor of sending scoring requests off to the server and using KataGo to do a presumably *much* better job than the score estimator, so wiring that up might be out of the scope of the goban project.